Jenkinsfile

```diff
@@ -162,18 +162,18 @@ pipeline {
         } // Standard agent tests
 
         stage('Azure Authenticator') {
+          agent { label 'azure-linux' }
+          environment {
+                   AZURE_AUTHN_INSTANCE_IP = sh(script: 'curl "http://checkip.amazonaws.com"', returnStdout: true).trim()
+                   SYSTEM_ASSIGNED_IDENTITY = sh(script: 'ci/authn-azure/get_system_assigned_identity.sh', returnStdout: true).trim()
+          }
           steps {
-            script {
-              node('azure-linux') {
-                // get `ci/authn-azure/get_system_assigned_identity.sh` from scm
-                checkout scm
```

Comment:
how come we don't need this?

Reply:
checkout scm is not needed when using agent